src/frontend/src/lib/components/ui/MessageBox.svelte

```diff
@@ -1,25 +1,44 @@
 <script lang="ts">
+	import { slide } from 'svelte/transition';
+	import IconClose from '$lib/components/icons/lucide/IconClose.svelte';
 	import IconInfo from '$lib/components/icons/lucide/IconInfo.svelte';
+	import { SLIDE_EASING } from '$lib/constants/transition.constants';
+	import { i18n } from '$lib/stores/i18n.store';
 
 	export let level: 'plain' | 'info' | 'light-warning' | 'error' = 'info';
+	export let closable = false;
```

Comment:
In `SendInfo` the `MessageBox` is already displayed or not, do you plan to refactor this afterwards?

Reply:
by default the `MessageBox` is not closable, so, in `SendInfo` case, the logic is based more on the context; and it is not allowed to be closable

I think I can leave it as it is, no? Meaning that if at some point we want to make it closable in that case too, we can change that logic

WDYT?

Reply:
Works for me for sure.